src/components/InLineNotification.tsx

```diff
@@ -116,8 +116,8 @@ const styles = StyleSheet.create({
 
 const severityColors: Record<Severity, CustomColors> = {
   [Severity.Informational]: {
-    primary: Colors.infoDark,
-    secondary: Colors.infoLight,
+    primary: Colors.black,
```

Comment:
Checked with Kayla and Nitya, they want this style updated to all InlineNotifications